src/tests/PGCustomInsertTest.ts

```diff
@@ -1,40 +1,42 @@
-import { UsageStats } from "../UsageStats";
+import {UsageStats} from "../UsageStats";
 
-const pgp = require('pg-promise')({ noWarnings: true});
+const pgp = require('pg-promise')();
 import {Document} from "../Document";
 
-export class PGCustomInsertTest {
+const db = pgp({
+    host: "localhost",
+    port: 5432,
+    user: "postgres",
+    password: "test",
+    database: "perf_test"
+});
+
+const cs = new pgp.helpers.ColumnSet(["id", "docId", "label", "context", "distributions:json", "date"],
+    {table: 'document'});
 
-	public static start(docs: Document[]) {
-		let db = pgp({
-			host: "localhost",
-			port: 5432,
-			user: "postgres",
-			password: "test",
-			database: "perf_test"
-		});
+export class PGCustomInsertTest {
 
-		const columnSet = pgp.helpers.ColumnSet([ "id", "docId", "label", "context", "distributions:json", "date"],
-                                                    { table: 'document'});
+    public static start(docs: Document[]) {
 
-		let start = new Date().getTime();
+        const start = Date.now();
         const us = new UsageStats();
         us.start();
 
-		const queryString = pgp.helpers.insert(docs, columnSet);
-		return db.query(queryString)
-                .then(() => {
-                    let end = new Date().getTime();
-                    const stats = us.stop();
+        const sql = pgp.helpers.insert(docs, cs); // No conflict resolution? - see the other PR I did ;)
```

Comment:
Conflict resolution should not be needed, since all id's are guaranteed to be unique. Could be interesting to implement another test suite to also test that.